README.md

```diff
@@ -1 +1,9 @@
 # `docker-capo-controller-manager`
+This is a base image build on top of `docker-ubuntu` which can
```

Comment:
This will need to be revised and please add a new line above it.

Reply:
done